### PR TITLE
Rewrite Pterodactyl integration

### PR DESCRIPTION
**Pull request by Telesphoreo**
- Rewrite Pterodactyl integration
- Bring back the restart command and fix the stop command to gracefully stop the server now
- I had to update for Java 16, sorry. I can't get it to compile on 11 anymore anyway

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -253,18 +263,38 @@
             <scope>compile</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
```
Comment (VideoGameSmash12) [inline pom.xml:268]: any particular reason this was re-added?

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_restart.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        RESTART_CONFIRM.put(sender, reason);
+        new BukkitRunnable()
+        {
+            @Override
+            public void run()
+            {
+                if (RESTART_CONFIRM.containsKey(sender))
+                {
+                    RESTART_CONFIRM.remove(sender);
+                    msg("Stop request expired.");
```
Comment (VideoGameSmash12) [inline src/main/java/me/totalfreedom/totalfreedommod/command/Command_restart.java:58]: i know this is such a nitpick but change "stop" to "restart"

**Review by VideoGameSmash12 — COMMENTED** [review]
just some nitpicks

File: pom.xml (hunk 1 above)
Comment (Telesphoreo) [reply pom.xml:268]: It doesn't compile without it, try removin git

File: src/main/java/me/totalfreedom/totalfreedommod/command/Command_restart.java (hunk 2 above)
Comment (Telesphoreo) [reply src/main/java/me/totalfreedom/totalfreedommod/command/Command_restart.java]: Oops

**Review by Focusvity — CHANGES_REQUESTED** [review]
Can you please change the java version back to 11 and remove Lombok. This will not be merged into development (going into a branch) as I will be testing it.

**Comment by Telesphoreo** [comment]
I can but I'm not able to compile anymore on Java 11. I spent like 2 hours getting Java 16 as my version so I could run 1.17 and am not changing it back for one commit for you. You'll have to sure it compiles on your own

**Review by Focusvity — APPROVED** [review]

**Pull request merged**